src/mixin/describable-entity-mixin.js

```diff
@@ -0,0 +1,27 @@
+import { performSirenAction } from '../es6/SirenAction.js';
+
+export const DescribableEntityMixin = superclass => class extends superclass {
+
+	_isDescribableEntity() {
```

Comment:
I think it makes sense to make these checks public. The reasoning is that it lets the component thats using it decide whether the mixin behavior is optional or raise an error of some form if its required for the component to function.